src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  font-size: 13px;
+  font-family: Arial, sans-serif;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+h2 {
+  font-size: 15px;
+  margin: 1em 0;
+  font-weight: normal;
+}
+
+p {
+  margin: 0;
+}
```

```suggestion
h1,
p {
  margin: 0;
}
```

You have `margin: 0` in your `h1` and `p` selectors bots. Dont repeat yourself. You may write like I've shown above and remove `margin: 0` from your `h1` rules